generated/nifake/nifake/_library.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, ctypes_library):
         self._library = ctypes_library
         # We cache the cfunc object from the ctypes.CDLL object
         self.niFake_Abort_cfunc = None
+        self.niFake_AcceptListOfTimeValues_cfunc = None
```

Comment:
Are these in internal NI metadata? Please confirm.

Reply:
They are in a changelist for now. Once I get this merged, I'll send a review for that.